kingfisher_scrapy/extensions.py

```diff
@@ -198,7 +198,10 @@ def item_scraped(self, item, spider):
 
         if isinstance(item, FileItem):
             data['number'] = item['number']
-            data['data'] = item['data']
+            if isinstance(item['data'], str):
```

Comment:
Would this work, too?

```suggestion
            if isinstance(item['data'], (bytes, str)):
```